Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -79,10 +80,11 @@ final class PersonCell: UICollectionViewCell {
     func configure(with person: Person) {
         let df = DateFormatter()
         df.dateFormat = "dd.MM.yyyy"
-        
-        profileImageView.image = UIImage(named: "man-in-red-jacket-1681010")
-        // ^Update to default place holder image when resource is added.
-        // TO-DO: Use Kilo's ImageDownloader here to DL image
+        profileImageView.kf.indicatorType = .activity
+        profileImageView.kf.setImage(
+          with: URL(string: person.media[0]), //TODO: Add thumbnail to person model?
```

Comment:
This seems dangerous because we can end up with an index out of range error if there aren't any media for this person. There are a couple ways to fix this; here's one:
```suggestion
          with: URL(string: person.media.first ?? ""), //TODO: Add thumbnail to person model?
```

Reply:
if I'm not mistaken this can be a linter warning. Might be something to look into.